packages/next/src/server/config.ts

```diff
@@ -676,14 +676,19 @@ function assignDefaults(
       ...userProvidedOptimizePackageImports,
       'lucide-react',
       '@headlessui/react',
-      '@fortawesome/fontawesome-svg-core',
-      '@fortawesome/free-solid-svg-icons',
       '@headlessui-float/react',
       '@heroicons/react/20/solid',
       '@heroicons/react/24/solid',
       '@heroicons/react/24/outline',
       '@visx/visx',
       '@tremor/react',
+      'rxjs',
+      '@mui/material',
```

Comment:
question: could we hit the same issue as mentioned here?

https://github.com/vercel/next.js/pull/51953

Reply:
@OlegLustenko `optimizePackageImports` works differently from `modularizeImports`, this new option is automatic and works for all packages/versions out of the box (introduced to solve exactly that issue you linked). More details: https://github.com/vercel/next.js/pull/54695